charts/hono/templates/hono-adapter-amqp/hono-adapter-amqp-deployment.yaml

```diff
@@ -16,6 +16,7 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   {{- include "hono.metadata" $args | nindent 2 }}
+  {{- include "hono.deploymentAnnotations" $args | nindent 2 }}
```

Comment:
How about using `indent` instead of `nindent` to omit the added empty line here?

Reply:
yeah sure, updated